Volume/Networking/schema.json

```diff
@@ -0,0 +1,1598 @@
+{
```

Comment:
and let's not commit this file too! instead, we should add instructions in the README for which command to run in terminal to download this file (like eatery's readme) and have devs keep a local copy